File: 백준/Bronze/10699. 오늘 날짜/오늘 날짜.go
package main

import (
	"fmt"
	"strconv"
	"time"
)

func main() {
	t := time.Now()

	var yyyy, mm, dd string
	y, m, d := t.Date()
	yyyy = dateformat(y)
	mm = dateformat(int(m))
	dd = dateformat(d)

	fmt.Printf("%s-%s-%s", yyyy, mm, dd)
}

func dateformat(i int) string {
	var o string
	if i >= 10 {
		o = strconv.Itoa(i)
	} else {
		o = "0" + strconv.Itoa(i)
	}
	return o
}

File: 백준/Bronze/10699. 오늘 날짜/오늘 날짜.java
import java.text.DateFormat;
import java.text.SimpleDateFormat;
import java.util.Date;

public class Main {
    public static void main(String[] args) {
        Date date = new Date();
        DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
        System.out.println(dateFormat.format(date));
    }
}